hyper/http20/connection.py

```diff
@@ -624,6 +631,21 @@ def _recv_cb(self):
             except ConnectionResetError:
                 break
 
+    def _send_rst_frame(self, stream_id, error_code):
+        """
+            Send reset stream frame with error code and remove stream from map.
+        """
+        if error_code is not None:
+            f = RstStreamFrame(stream_id)
+            f.error_code = error_code
+            self._send_cb(f)
```

Comment:
I don't think we need to cover the `error_code is None` case, that's surely an error.